invenio_app_rdm/theme/views.py

```diff
@@ -62,11 +64,20 @@ def init_menu():
 #
 # Views
 #
+
+
 def index():
     """Frontpage."""
+    featured_communities = current_communities.service.featured_search(
```

Comment:
what if an instance will not have feature communities in the home page? This extra search should be avoided in this case.

Reply:
Plus one on this!